content/ja/docs/concepts/overview/components.md

```diff
@@ -105,11 +102,11 @@ Kubernetesによって開始されたコンテナは、DNS検索にこのDNSサ
 
 ### コンテナリソース監視
 
-[コンテナリソース監視](/docs/tasks/debug-application-cluster/resource-usage-monitoring/)は、コンテナに関する一般的な時系列メトリックを中央データベースに記録します。また、そのデータを閲覧するためのUIを提供します。
+[コンテナリソース監視](/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/)は、コンテナに関する一般的な時系列メトリックを中央データベースに記録します。また、そのデータを閲覧するためのUIを提供します。
 
-### クラスターレベルログ
+### クラスターレベルのロギング
 
-[クラスターレベルログ](/docs/concepts/cluster-administration/logging/)メカニズムは、コンテナのログを、検索／参照インターフェイスを備えた中央ログストアに保存します。
+[クラスターレベルのロギング](/ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/)メカニズムは、コンテナのログを、検索／参照インターフェイスを備えた中央ログストアに保存します。
```

Comment:
EN content has the URL "https://kubernetes.io/docs/concepts/cluster-administration/logging/" as link.
Why do you update this URL with "debug-cluster"?
```suggestion
[クラスターレベルのロギング](/ja/docs/concepts/cluster-administration/logging/)メカニズムは、コンテナのログを、検索／参照インターフェイスを備えた中央ログストアに保存します。
```